lib/presentation/feature/news/widgets/top_anime_item_widget.dart

```diff
@@ -1,39 +1,53 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
 
 import '../../../../constants/resources/colors.dart';
 import '../../../../constants/resources/images.dart';
-import '../models/anime_news_item_model.dart';
+import '../../../../constants/routes.dart';
+import '../models/top_anime_item_model.dart';
 
 class TopAnimeItemWidget extends ConsumerWidget {
   const TopAnimeItemWidget({
     required this.animeNewsItemModel,
     Key? key,
   }) : super(key: key);
-  final AnimeNewsItemModel animeNewsItemModel;
+  final TopAnimeItemModel animeNewsItemModel;
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    return GridTile(
-      footer: Material(
-        color: Colors.transparent,
-        shape: const RoundedRectangleBorder(
-          borderRadius: BorderRadius.vertical(bottom: Radius.circular(4)),
-        ),
-        clipBehavior: Clip.antiAlias,
-        child: GridTileBar(
-          backgroundColor: AppColors.light.mediumBlack,
-          title: _GridTitleText(animeNewsItemModel.titleEn),
-          subtitle: _GridTitleText(animeNewsItemModel.titleJp),
-        ),
+    return GestureDetector(
+      onTap: () => context.pushNamed(
```

Comment:
if we can use path only i would prefer to use context.go with extra method

Reply:
Ok this is fair, although I thought it would be nice to demonstrate how to use named Routes, since they will be easier to use in long paths. In fact, in this case, since it is only 1 parameter, we can use only the "extra" parameter, so I'll refactor it.

Reply:
I refactored, but before I commit it I would like to show you guys on Monday to get your opinion, because I don't think it's an improvement.